.github/workflows/localization_branch_sync.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync Localization Branch
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types: [closed]
+
+concurrency: 
+  group: 'Localization-Sync'
+  cancel-in-progress: false
+
+jobs:
+  Replace Localization Branch:
```

Comment:
I would remove the spaces and will just lowercase everything. You already have a human readable name in line 14.